frontend/app/src/components/accounts/manual-balances/ManualBalancesForm.vue

```diff
@@ -177,18 +171,10 @@ const openCustomAssetForm = async () => {
   set(showCustomAssetForm, true);
 };
 
-const saveCustomAsset = async () => {
-  set(customAssetFormSaving, true);
-
-  set(customAssetFormSaving, true);
-  const identifier = await get(customAssetFormRef)?.save();
-
-  if (identifier) {
-    set(showCustomAssetForm, false);
-    set(asset, identifier);
+const customAssetSaved = async (assetId: string) => {
+  if (assetId) {
```

Comment:
Do we emit falsy values from the components?

Reply:
no, my bad, the if is unnecessary.